From: Jason Rhinelander <jason@imaginary.ca>
Date: Mon, 6 Apr 2020 19:53:48 -0300
Subject: Update to lmq 1.0.5 & fix compilation

Updates to 1.0.5 with a fix that can cause message reception delays of
up to 250ms.

1.0.5 also broke SS's forward declaration of `string_view` (because it
grew the capability of handling char types other than `char`); the
underlying simple_string_view class is an implementation detail that
isn't meant to be used (and may not exist at all: lokimq uses
std::string_view instead if the compiler provides it).
---
 httpserver/lmq_server.h | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/httpserver/lmq_server.h b/httpserver/lmq_server.h
index 41bb805..0d1ac96 100644
--- a/httpserver/lmq_server.h
+++ b/httpserver/lmq_server.h
@@ -3,11 +3,10 @@
 #include <cstdint>
 #include <memory>
 #include <string>
+#include <lokimq/string_view.h>
 
 namespace lokimq {
 class LokiMQ;
-class simple_string_view;
-using string_view = simple_string_view;
 struct Allow;
 class Message;
 } // namespace lokimq
